src/Test.UnitTests.Sarif.Multitool/RebaseUriCommandTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Multitool;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
+{
+    public class RebaseUriCommandTests : FileDiffingUnitTests
```

Comment:
Wow, the rebaselineuri command had no tests before! Glad we're adding them.

Reply:
Si! i will further refine this in future to see if we can generalize this to run file diffing tests for all commands, i see this need. we dont have such tests for other commands that i have been using e.g. AbsoluteUri.

---
In reply to: [298673342](https://github.com/microsoft/sarif-sdk/pull/1549#discussion_r298673342) [](ancestors = 298673342)